augur/export_v2.py

```diff
@@ -1037,7 +1037,7 @@ def run_v2(args):
 
     # pass through any extensions block in the auspice config JSON without any changes / checking
     if config.get("extensions"):
-        data_json["extensions"] = config["extensions"]
+        data_json["meta"]["extensions"] = config["extensions"]
```

Comment:
Technically this is a breaking change which would prompt a major version bump, but if we're very sure that no one else is using this, maybe OK to be a bug fix instead?

Reply:
True, I don't mind either way - 14.1.0 was released around a week ago?
Up to you :)

Reply:
It's also no biggie to bump to version 15 and not have to hem and haw over if it's ok to break our rules this time or not. :-) Version numbers are cheap!